packages/gatsby-theme-blog-core/src/components/post.js

```diff
@@ -1,3 +1,21 @@
 import React from "react"
 
-export default props => <pre>{JSON.stringify(props.data, null, 2)}</pre>
+import { MDXRenderer } from "gatsby-plugin-mdx"
+
+import PostTitle from "./post-title"
+import PostDate from "./post-date"
+import PostHero from "./post-hero"
+
+const Post = ({data}) => {
+    const post = data.blogPost
+    return (
+        <main>
+        <PostHero post={post} />
+        <PostTitle>{post.title}</PostTitle>
```

Comment:
I'm guessing the reason for not keeping this markup structure all in the same place is to allow for finer-grained shadowing, right?

Reply:
Ya, that was the idea. And to match the structure in blog theme. Arguably we could collapse it. I can see both sides.